boards/microbit/Makefile.include

```diff
@@ -5,21 +5,15 @@ export CPU_MODEL = nrf51x22xxab
 PORT_LINUX ?= /dev/ttyACM0
 PORT_DARWIN ?= $(firstword $(sort $(wildcard /dev/tty.SLAB_USBtoUART*)))
 
-# this board supports flashing through plain fscopy, using JLink or using
-# openocd. Default programmer is fscopy
-PROGRAMMER ?= fscopy
+# for this board we support flashing via openocd or pyocd
+PROGRAMMER ?= openocd
 
-ifeq (fscopy,$(PROGRAMMER))
-  FFLAGS = $(HEXFILE)
-  DEBUGGER_FLAGS =
-
-  FLASHER = $(RIOTBOARD)/$(BOARD)/dist/flash.sh
-  DEBUGGER =
-  DEBUGSERVER =
-else ifeq (openocd,$(PROGRAMMER))
-  # this board uses a daplink adapter by default
+ifeq (openocd,$(PROGRAMMER))
   DEBUG_ADAPTER = dap
-else ifeq (pyocd, $(PROGRAMMER))
+else ifeq (pyocd,$(PROGRAMMER))
+  # PyOCD doesn't recognize automatically the board ID, so target type has to
```

Comment:
I think this is not true and not needed for microbit

Reply:
See [here](https://github.com/mbedmicro/pyOCD/blob/master/pyocd/board/board_ids.py#L193). But this is required for calliope-mini since it isn't in the list of known board in PyOCD.

Reply:
Ah, I see. I was already wondering about this. How about we make quick PR to PyOCD to add the calliope to the list also?